pytorch_lightning/logging/tensorboard.py

```diff
@@ -89,6 +95,17 @@ def save(self):
             # you are using PT version (<v1.2) which does not have implemented flush
             self.experiment._get_file_writer().flush()
 
+        # create a preudo standard path ala test-tube
+        dir_path = os.path.join(self.save_dir, self.name, 'version_%s' % self.version)
+        if not os.path.isdir(dir_path):
+            dir_path = self.save_dir
+        # prepare the file path
+        meta_tags_path = os.path.join(dir_path, self.NAME_CSV_TAGS)
+        # save the metatags file
+        df = pd.DataFrame({'key': list(self.tags.keys()),
```

Comment:
Pandas is a really heavy dependency to pull in just for writing a few lines of CSV. Why not just use the built-in csv module?

Reply:
good point... this was WIP to find the issue and patch it...

Reply:
well, Pandas is already in `core.memory` and `trainer.training_io` so?

Reply:
Fair. I'd prefer to get rid of it, but we can deal with it later.

Reply:
sure, I do agree, but then make another PR just with removing Pandas...